docs/design/features/globalization-icu.md

```diff
@@ -0,0 +1,131 @@
+# .NET Globalization ICU on all runtimes
+
+Author: [Santiago Fernandez Madero](https://github.com/safern)
+
+In .NET Core our globalization APIs behavior used different underlying libraries, in Unix, we used [International Components for Unicode (ICU)](http://site.icu-project.org/home) and in Windows, we used [National Language Support (NLS)](https://docs.microsoft.com/en-us/windows/win32/intl/national-language-support). This caused some behavior differences in a handful number of Globalization APIs:
+
+- Cultures and culture data
+- String casing
+- String sorting and searching
+- Sort keys
+- String Normalization
+- Internationalized Domain Names (IDN) support
+- Time Zone display name on Linux
+
+## ICU on Windows
+
+Windows 10 May 2019 Update and later versions started including [icu.dll](https://docs.microsoft.com/en-us/windows/win32/intl/international-components-for-unicode--icu-) as part of the OS, so in .NET 5.0 we changed the default behavior for Globalization APIs when running in Windows, which tries to load `icu.dll` and if we can load it, we use ICU as the underlying native APIs, if we fail to load it, we use the legacy behavior (NLS), this will be the case for older versions of Windows.
```

Comment:
```suggestion
Windows 10 May 2019 Update and later versions include [icu.dll](https://docs.microsoft.com/en-us/windows/win32/intl/international-components-for-unicode--icu-) as part of the OS, and .NET 5 now uses that by default. When running on Windows, .NET 5 tries to load `icu.dll` and uses it for the globalization implementation if it's available.  If that library can't be found or loaded, such as when running on older versions of Windows, .NET 5 will fall back to NLS-based implementation.
```

Reply:
> Windows 10 May 2019 Update and later versions include [icu.dll](https://docs.microsoft.com/en-us/windows/win32/intl/international-components-for-unicode--icu-) as part of the OS, and .NET 5 

Windows started including the ICU from RS2 but we'll have the minimum requirements of the version shipped in 19H1.